addon/package.json

```
@@ -9,11 +9,32 @@
   "repository": "https://github.com/ember-modifier/ember-modifier",
   "license": "MIT",
   "author": "",
-  "types": "index.d.ts",
+  "exports": {
+    ".": "./dist/index.js",
+    "./*": "./dist/*",
+    "./addon-main.js": "./addon-main.js"
+  },
+  "typesVersions": {
+    "*": {
+      ".": [
+        "./dist/index.d.ts"
+      ],
+      "*": [
+        "./dist/*"
+      ],
+      "./-private/signature": [
+        "./dist/-private/signature.d.d.ts"
```

This should be `.d.ts` not `.d.d.ts`. Again, not blocking for *this* PR, but we should also confirm after landing it that we should be able to remove this mapping entirely and have things still work.

```suggestion
        "./dist/-private/signature.d.ts"
```